app/sample_manifest_excel/sample_manifest_excel/worksheet/test_worksheet.rb

```diff
@@ -13,7 +13,6 @@ class TestWorksheet < SequencescapeExcel::Worksheet::Base # rubocop:todo Metrics
 
```

Comment:
The rules for accessioning are painfully complicated, and depend on the study. Updated this to let my inject my own studies.